README.md

```diff
@@ -99,15 +99,22 @@ You can run commands in every javascript project using Nx: `nx run-many --target
 This allows you to lint and/or test all projects at once.
 
 ### Publishing a Release
-To publish a release, use lerna to create new versions of all changed packages (ensure you do this on the main branch)
+Setup:
+- run `aws sso login`
+- run `./scripts/brew-install.sh`
+- ensure you're on the main branch with the latest code 
+ 
+Then use lerna to create new versions of all changed packages (ensure you do this on the main branch)
 `yarn lerna:version`
 
-Push up the new tags and version changes, then run:
+This will automatically push the latest tags and version updates to github. 
 
+To publish the versions to npm, you need a one-time password from our NPM account. Search for 'npm' in 1Password and copy the OTP under `Npmjs (taplyticsuser)`
```

Comment:
I think moving this line up here is a great improvement (nothing like instructions that say "run a command, but before you do that..." after you run the command :P ) but can drop the reference to 1pass